packages/puppeteer-core/src/cdp/CdpBluetoothEmulationManager.ts

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
Add export in `cdp.ts`.

Reply:
Also I think filename should be without the Cdp prefix for consistency